### ISSUE TITLE
Starting a blog - Comments

### ISSUE THREAD

**Issue by farre** [farre/blog#1]
This issue is reserved for comments on the blog post [Starting a blog](https://farre.github.io/blog/posts/starting-a-blog.html).
Leave a comment below and it will show up on the blog post's Web page. Thanks.

**Comment by farre** [comment]
A blog should be able to support comments.

**Comment by farre** [comment]
And of course several comments.

**Comment by mo** [comment]
Feature request: RSS

**Comment by farre** [comment]
Yep, [this](https://jaspervdj.be/hakyll/tutorials/05-snapshots-feeds.html) will be added soon. But I had to rush a bit since I need a blog to write about stuff sooner :)

And I guess some kind of twitter integration would be nice. But I'm really happy with the GitHub issues for comments. Owe @izuzak for [this idea](http://ivanzuzak.info/2011/02/18/github-hosted-comments-for-github-hosted-blogs.html).
